HybridMiningStrategyIT fails after updating to Kafka 4.2

## Bug report

After updating to Kafka 4.2, the `HybridMiningStrategyIT` now fails 

```
20:02:41,618 [ERROR] io.debezium.connector.oracle.logminer.buffered.HybridMiningStrategyIT.shouldStreamSchemaChangeWithExistingLegacySchemaHistory -- Time elapsed: 0.216 s <<< ERROR!
org.apache.kafka.common.config.ConfigException: Missing required configuration "bootstrap.servers" which has no default value.
	at org.apache.kafka.common.config.ConfigDef.parseValue(ConfigDef.java:537)
	at org.apache.kafka.common.config.ConfigDef.parse(ConfigDef.java:527)
	at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:118)
	at org.apache.kafka.connect.runtime.WorkerConfig.<init>(WorkerConfig.java:449)
	at io.debezium.connector.oracle.logminer.buffered.HybridMiningStrategyIT$TestWorkerConfig.<init>(HybridMiningStrategyIT.java:1468)
	at io.debezium.connector.oracle.logminer.buffered.HybridMiningStrategyIT.createOffsetBasedOnCurrentScn(HybridMiningStrategyIT.java:1222)
	at io.debezium.connector.oracle.logminer.buffered.HybridMiningStrategyIT.shouldStreamSchemaChangeWithExistingLegacySchemaHistory(HybridMiningStrategyIT.java:815)

20:02:41,618 [ERROR] io.debezium.connector.oracle.logminer.buffered.HybridMiningStrategyIT.shouldStreamOfflineSchemaChangeWithExistingLegacySchemaHistory -- Time elapsed: 0.204 s <<< ERROR!
org.apache.kafka.common.config.ConfigException: Missing required configuration "bootstrap.servers" which has no default value.
	at org.apache.kafka.common.config.ConfigDef.parseValue(ConfigDef.java:537)
	at org.apache.kafka.common.config.ConfigDef.parse(ConfigDef.java:527)
	at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:118)
	at org.apache.kafka.connect.runtime.WorkerConfig.<init>(WorkerConfig.java:449)
	at io.debezium.connector.oracle.logminer.buffered.HybridMiningStrategyIT$TestWorkerConfig.<init>(HybridMiningStrategyIT.java:1468)
	at io.debezium.connector.oracle.logminer.buffered.HybridMiningStrategyIT.createOffsetBasedOnCurrentScn(HybridMiningStrategyIT.java:1222)
	at io.debezium.connector.oracle.logminer.buffered.HybridMiningStrategyIT.shouldStreamOfflineSchemaChangeWithExistingLegacySchemaHistory(HybridMiningStrategyIT.java:722)
```

This is because `WorkerConfig` in Kafka 4.2 no longer specifies a default value for `bootstrap.servers`.

--- a/debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/buffered/HybridMiningStrategyIT.java
+++ b/debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/buffered/HybridMiningStrategyIT.java
@@ -1216,6 +1216,7 @@ public class HybridMiningStrategyIT extends AbstractAsyncEngineConnectorTest {
         embeddedConfig.put(StandaloneConfig.OFFSET_STORAGE_FILE_FILENAME_CONFIG, OFFSET_STORE_PATH.toAbsolutePath().toString());
         embeddedConfig.put(WorkerConfig.KEY_CONVERTER_CLASS_CONFIG, keyConverter.getClass().getName());
         embeddedConfig.put(WorkerConfig.VALUE_CONVERTER_CLASS_CONFIG, valueConverter.getClass().getName());
+        embeddedConfig.put(WorkerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
         System.out.println(embeddedConfig);
 
         final OffsetBackingStore store = KafkaConnectUtil.fileOffsetBackingStore();

